db/connectToMongoose.js

```diff
@@ -0,0 +1,24 @@
+import mongoose from "mongoose";
+
+export function connectToMongoose(database_address) {
+    if (
+        database_address !== undefined &&
+        process.env.FUNCTIONAL_TESTS !== "true"
+    )
+        connMongoose(database_address);
+}
```

Comment:
it would be nice to get info that you didn't connect to database here, rather than during database call, e.g.:
```suggestion
    if (
        database_address !== undefined &&
        process.env.FUNCTIONAL_TESTS !== "true"
    )
        connMongoose(database_address);
    else {
        console.log("Didn't connect to database.")
    }
}
```